.docs/content/1.concepts/4.components.md

```diff
@@ -0,0 +1,75 @@
+
+
+# Project components
+
+This page intends to describe the different software projects and components used for ArmoniK
+internals. To learn how to install or use ArmoniK, please see these repositories:
+  * install:
+  * use:
+
+
+## Common Library
+
+**ArmoniK.Common** provides the components required by all the other compononents of ArmoniK.
+
+## Control
+
+Different projects are available in the control plan:
+
+* **ArmoniK.Control.Submitter** is a web server providing the gRPC services required by the
+SubmitterService client from SDK. It provides the following services:
+  * Session creation
+  * Task submission
+  * Results retrieving
+  * Task cancellation
+  * Session cancellation
+
+* **ArmoniK.Control.Monitor (not yet available)** is a web server providing the gRPC services
+required by the MonitorService client from SDK. It provides the followin services:
+  * List all session and tasks
+  * Count the session and task per status
+  * Get the execution details for the tasks
+
+* **ArmoniK.Control.ResourceManager (not yet available)** is a web server providing gRPC
+services required by the ResourceManager client from SDK. It provides the following services:
+  * Upload new resources
+  * Download new resources
+  * List all available resources
+  * Delete resources
+
+* **ArmoniK.Control.Autoscaling (not yet available)** is a process that regularly computes
+metrics used to determine the number of POD instances required on the grid. This will then
+be used by Kubernetes to start/stop compute nodes (on elastic configurations such as managed
+K8s service on cloud).
+
+## Compute Plan Components
+
+From a software enginnering point of view, the compute plan relies on three components:
+
+* **ArmoniK.Compute.PollingAgent** implements a ArmoniK.DevelopmentKit.Gridlib.GRPC client.
+It acts as a proxy between the gridlib agent container and the rest of the ArmoniK system.
+Using such a proxy agent allows all the ArmoniK logic to be implemented independantly of
+the Gridlib agent. Hence handling an new agent to handle new languages will be easier.
+
+* Other companion containers for the gridlib agent container Such cantainers could provide
```

Comment:
This sentence is not clear to me, do you mean that this companion containers providing these functionalities are optional?

Reply:
This file is the old readme. I moved it to avoid losing it. It is still outdated and needs to be changed.